warp10/src/main/java/io/warp10/continuum/store/Store.java

```diff
@@ -1011,6 +1013,25 @@ public BulkDeleteResponse call(BulkDeleteService service) throws IOException {
       Sensision.update(SensisionConstants.SENSISION_CLASS_CONTINUUM_STORE_HBASE_DELETE_OPS, Sensision.EMPTY_LABELS, 1);
       Sensision.update(SensisionConstants.SENSISION_CLASS_CONTINUUM_STORE_HBASE_DELETE_REGIONS, Sensision.EMPTY_LABELS, noOfRegions);
       Sensision.update(SensisionConstants.SENSISION_CLASS_CONTINUUM_STORE_HBASE_DELETE_DATAPOINTS, Sensision.EMPTY_LABELS, noOfDeletedVersions);
+
+      Metadata meta = msg.getMetadata();
+      if (null != meta) {
+        Map<String, String> labels = new HashMap<>();
+
+        String owner = meta.getLabels().get(Constants.OWNER_LABEL);
```

Comment:
owner and application cannot be absent from metadata, so these tests are superfluous and can be removed.